themes/hugo_theme_robust/static/css/styles.css

```diff
@@ -600,7 +661,7 @@ article.single .body p>code {
   color: #78909c;
 }
 
-article.single .body h1 { margin: 2rem 0; font-weight: 700; }
+article.single .body h1 { margin: 2rem 0; font-weight: 700; clear:both; }
```

Comment:
clear only for h1?  If this is a good idea it should probably be put on them all.